src/Worker/DiscoverPoCo.php

```diff
@@ -164,13 +164,8 @@ private static function discoverUsers() {
 				continue;
 			 }
 
-			if (in_array($urlparts["host"], ["www.facebook.com", "facebook.com", "twitter.com",
-								"identi.ca", "alpha.app.net"])) {
-				$networks = ["www.facebook.com" => NETWORK_FACEBOOK,
-						"facebook.com" => NETWORK_FACEBOOK,
-						"twitter.com" => NETWORK_TWITTER,
-						"identi.ca" => NETWORK_PUMPIO,
-						"alpha.app.net" => NETWORK_APPNET];
```

Comment:
You removed NETWORK_APP too, was it on purpose?

Reply:
app.net is dead for several years.